objects/Handlebars.js

```diff
@@ -22,26 +22,25 @@ export default class Handlebars extends CustomObject {
         this.simulateStop = false;
     }
 
-    render(context, program_state, t, a, simulateStop, model_transform=Mat4.identity()) {
+    render(context, program_state, t, a, simulateStop, num_handlebars = 15, spacing = 3.5, bar_length = 20, model_transform=Mat4.identity()) {
```

Comment:
Rather than hard coding spacing (which then we need to update every time we change the number of handle bars), we can just have this class dynamically calculate the number of handle bars needed. So we can remove this parameter. 

```suggestion
    render(context, program_state, t, a, simulateStop, num_handlebars = 15, bar_length = 20, model_transform=Mat4.identity()) {
```